config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml

```diff
@@ -230,14 +230,15 @@ periodics:
     testgrid-dashboards: sig-windows-master-release, sig-windows-signal
     testgrid-tab-name: capz-windows-containerd-master-serial-slow-hpa
 - name: ci-kubernetes-e2e-capz-master-windows-2022
+  cluster: k8s-infra-prow-build
   interval: 3h
   decorate: true
   decoration_config:
     timeout: 4h
   labels:
     preset-dind-enabled: "true"
     preset-kind-volume-mounts: "true"
-    preset-azure-cred-wi: "true"
+    preset-azure-community: "true"
     preset-azure-anonymous-pull: "true" # Sets REGISTRY which is needed when building CCM/CNM images
```

Comment:
we will need to remove this if we add Registry to the preset.

Reply:
For windows we don't need it, but we might need to create a different preset for CAPZ which uses `USE_LOCAL_KIND_REGISTRY` which is in that preset

Reply:
@jackfrancis

Reply:
based on this

```
  - name: REGISTRY
    value: "capzcicommunity.azurecr.io"
```

yeah it seems like we should include `USE_LOCAL_KIND_REGISTRY=false` in the preset

Is that what you mean?